ExampleGenerator/Series/StemSeriesExamples.cs

```diff
@@ -0,0 +1,88 @@
+﻿namespace ExampleGenerator
+{
+	using System;
+	using OxyPlot;
+	using OxyPlot.Axes;
+	using OxyPlot.Series;
+
+	public class StemSeriesExamples
+	{
+
+		[Export(@"Series\StemSeries")]
+		public static PlotModel StemSeries()
```

Comment:
I think the basic example needs just a single series. Could also use default values for the marker style, I think.